pages/_document.page.js

```diff
@@ -35,6 +35,9 @@ export default function Document() {
         <link rel="stylesheet" href="/assets/css/nice-select.css" />
         <link rel="stylesheet" href="/assets/css/style.css" />
 
+        <link rel="stylesheet" href="/assets/css/footer.css" />
+
+
```

Comment:
ohh no no! create css file in styles folder. see hackathon.css

Reply:
ok sure